### ISSUE TITLE
macOS: Cannot open file from explorer with keyboard

### ISSUE THREAD

**Issue by JacksonKearl** [Microsoft/vscode#53547]
Issue Type: <b>Bug</b>

In Windows, `Enter` opens a selected file from Explorer. In Mac, `Enter` renames the selected file. This is probably to mirror the default Finder behavior. There does not seem to be a way to open. I think `Space` would be good. The defualt Finder behavior for `Space` is to open a preview. 

VS Code version: Code - Insiders 1.25.0-insider (0f080e5267e829de46638128001aeb7ca2d6d50e, 2018-07-04T14:06:34.850Z)
OS version: Darwin x64 17.5.0

**Comment by bpasero** [comment]
`Cmd+ArrowDown`
